docs/notices.md

```diff
@@ -2,14 +2,14 @@
 
 To add a notice:
 ```
-npm run generateNotice
+yarn generateNotice
```

Comment:
This file should be deleted in a separate PR as well - it was removed in https://github.com/MetaMask/metamask-extension/commit/ac8b56a00defff4cb44a6a34251a19d8ab6159b6